### PR TITLE
Fix/warning settings and fixes

### PR DESCRIPTION
**Pull request by TroubleMakerBen**

### CODE REVIEW

File: Global.xcconfig [hunk 1]
```diff
@@ -53,4 +52,10 @@ GCC_PREPROCESSOR_DEFINITIONS = $(inherited) $(XCODEBUILD_GCC_PREPROCESSOR_DEFINI
 // -padded
 // This isn’t really an issue to us, since we’re not programming embedded systems.
 
-WARNING_CFLAGS = -Weverything -Wno-objc-missing-property-synthesis -Wno-float-equal -Wno-pedantic -Wno-padded  -Wno-sign-conversion
+// -Wc++98
```
Comment (jaelim-ms) [inline Global.xcconfig]: -c++98-compat?

Comment (TroubleMakerBen) [inline Global.xcconfig]: Yes, if we don't add `-Wno-c++98-compat`, the compiler will ommit warnings that we use C++11 features. We're deliberately using C++11 and don't care about C++98 ;)

Comment (TroubleMakerBen) [inline Global.xcconfig]: Oooooh, I now get what you are saying. I'll update the comment 🙈

Comment (jaelim-ms) [inline Global.xcconfig]: 😄

**Pull request merged**